### PR TITLE
UI-III Completed Project Elijah McKay 

### PR DESCRIPTION
**Pull request by ElijahMcKay**

### CODE REVIEW

File: great-idea/css/index.css [hunk 1]
```diff
@@ -63,4 +63,135 @@ h1, h2, h3, h4, h5 {
...
+footer {
+	margin-top: 50px; 
+}
+
+.copy {
+	text-align: center; 
+}
+
+
+
```
Comment (AdetunjiShennaike) [inline great-idea/css/index.css:199]: great class names, and looks like you have a good understanding of flexbox

File: great-idea/css/services.css [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    list-style-type: disc;
+    margin-left: 20px; 
+}
+
+.copyright {
+    text-align: center; 
+}
+
+
+
```
Comment (AdetunjiShennaike) [inline great-idea/css/services.css:195]: you can actually have this in the same file due to your great class names, and if the html pages are using the same file, you wouldn't have to redo some things like the copyright, footer, header

File: great-idea/index.html [hunk 3]
```diff
@@ -17,7 +17,68 @@
...
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+      </nav>
+    </header>
+   
+    <div class="main-top">
+      <section class="main-left">
+        <h1>Innovation<br>On<br>Demand</h1>
+
```
Comment (AdetunjiShennaike) [inline great-idea/index.html:35]: good use of breaks

File: great-idea/index.html [hunk 4]
```diff
@@ -17,7 +17,68 @@
...
+    <hr>
+
+    <footer>
+      <h4>Contact</h4>
+        <p>123 Way 456 Street<br>
+           Somewhere, USA<br><br>
+           1 (888) 888-8888<br><br>
+           sales@greatidea.io<br><br><br></p>
+
+      <p class="copy">Copyright Great Idea! 2018</p>
```
Comment (AdetunjiShennaike) [inline great-idea/index.html:80]: fantastic job with keeping everything separated

**Review by AdetunjiShennaike — APPROVED** [review]

File: great-idea/css/services.css (hunk 2 above)
Comment (ElijahMcKay) [reply great-idea/css/services.css:195]: Thank you.  I will go back through and consolidate classes.

**Pull request merged**